data_generation/submit_tasks.py

```diff
@@ -6,13 +6,20 @@
 import shutil
 import json
 
+import random
+
 import inductiva
 
 FLAGS = flags.FLAGS
 
 flags.DEFINE_string("input_dataset", None, "Path to the dataset of objects.")
 
-flags.DEFINE_list("flow_velocity", [30, 0, 0], "Flow velocity in.")
+flags.DEFINE_list("flow_velocity_range_x", [30, 50],
+                  "Range of flow velocity in the x-direction.")
+flags.DEFINE_list("flow_velocity_range_y", [30, 50],
```

Comment:
Maybe I would set defaults of [0, 0] for y and z directions? You can always choose to set other values when you run the script... But defaulting those ranges in the FLAGS seems a bit unexpected, no? For x it's maybe ok... 

note: the other option is to not have any defaults at all and require the user to specify the values -- defaults make sense when it's safe to not specify those flags at all, and the main sources of variation are controlled by other FLAGS...